lucille-core/src/main/java/com/kmwllc/lucille/stage/EmitNestedChildren.java

```diff
@@ -36,6 +41,26 @@ public Iterator<Document> processDocument(Document doc) throws StageException {
 
     doc.setDropped(dropParent);
     List<Document> children = doc.getChildren();
+
+    // copy fields from parent to children
+    if (!fieldsToCopy.isEmpty()) {
+      for (Document child : children) {
+        for (String field : fieldsToCopy) {
+          if (!doc.has(field)) {
+            log.warn("Field '{}' not found in parent document", field);
+            continue;
+          }
+          // handle multivalued fields if needed, we want to be a list if multivalued
+          // Note we are grabbing the String version of the contents
+          if (doc.isMultiValued(field)) {
+            child.update(field, updateMode, doc.getStringList(field).toArray(new String[0]));
```

Comment:
can we do this in a way that preserves the type of the values and doesn't convert to String?

Reply:
resolved in [0c58d17](https://github.com/kmwtechnology/lucille/pull/225/commits/0c58d1712c7de03e9789791db3e64eba527e0fe1)